source/PlayerInfo.cpp

```diff
@@ -1,5 +1,5 @@
 /* PlayerInfo.cpp
-Copyright (c) 2014 by Michael Zahniser
+Copyright (c) 2020 by Michael Zahniser
```

Comment:
This means the first publish year.
You should not remove 2014, but you can add your copyright notice for your parts.

Reply:
ah right, does that mean I should add these files and my name to copyright.txt as well?

Reply:
IMO we have no need to add a copyright notice when modifying an existing source file because the contributions are recorded in the git-log and changelog file, but you can add your copyright notice in copyright file and here if you want to claim your copyright explicitly.

No one has added a copyright notice to the existing source files so far.